src/client/components/Sidebar/InterestingPeopleWithAPI.js

```diff
@@ -7,9 +7,9 @@ import User from './User';
 import Loading from '../../components/Icon/Loading';
 import steemAPI from '../../steemAPI';
 import './InterestingPeople.less';
+import './SidebarContentBlock.less';
 
-@withRouter
-class InterestingPeopleWithAPI extends Component {
+@withRouter class InterestingPeopleWithAPI extends Component {
```

Comment:
Can we have annotation on different line and extend from `React.Component`?

Reply:
yeah lol, forgot this was old stuff that got merged in, will update now

Reply:
We should have eslint for this stuff :P